packages/kit/src/module/install.ts

```diff
@@ -48,9 +48,8 @@ export const normalizeModuleTranspilePath = (p: string) => {
   return p.split('node_modules/').pop() as string
 }
 
-async function normalizeModule (nuxtModule: string | NuxtModule, inlineOptions?: any) {
-  const nuxt = useNuxt()
-
+export async function loadNuxtModuleInstance (nuxtModule: string | NuxtModule, nuxt: Nuxt = useNuxt()) {
```

Comment:
I renamed this and changed the signature:
- the name didn't really describe what was happening
- the inlineOptions was not accessed at all
- nuxt instance was recreated